demo/ZeroToHeroUsingTheBasis.md

```diff
@@ -0,0 +1,357 @@
+
+Go to http://test.bcovrin.vonx.io/, input the info from the response above.
+Then go and assign the did you have created, and put on the ledger, as public: `POST /wallet/did/public` [Localhost link to API doc](http://localhost:5000/api/doc#/wallet/post_wallet_did_public)
+
+**A public did we have!**
+This can be done programatically and all, and is very dependent on the ledger you will be using.
+
+All the next steps has to be done once, for each credential schema type. But once done for one credential type, you can issue as many as you want on that schema.
+
+## Creating the schema
+A schema is needed to make sure you know the attributes of a credential. This defines the names of the attributes and the schema, and also the version of the schema. Which is used in the credential definition
+[Localhost link to API doc](http://localhost:5000/api/doc#/schema/post_schemas)
+
+### Questions
+Are there any work going on in defining types in the schema or what is the 2 cents there?
+
+## Creating the credential definition 
+This is where you define if you want revocation or not. It also where you define the definition of the credential, based on a schema. [Localhost link to API doc](http://localhost:5000/api/doc#/credential-definition/post_credential_definitions)
+
+### Questions
+Why is this needed, except for revocation?
+
+## Verify
+By following this, you can verify that all is anchored to the ledger: https://github.com/hyperledger/aries-cloudagent-python/blob/main/demo/AriesOpenAPIDemo.md#confirming-your-schema-and-credential-definition
+
+# Issuance itself
+We will support ourselves on this: https://github.com/hyperledger/aries-cloudagent-python/blob/main/demo/AriesOpenAPIDemo.md#issuing-a-credential
+Just add any extra info that is not clear to a totally new person.
+We will go through both issuance v1.0 and v2.0 in the following text. Just to get the gist of the difference
+
+Issuance goes through a protocol of steps that is important. The steps are lightly explained here: https://github.com/hyperledger/aries-cloudagent-python/blob/main/demo/AriesOpenAPIDemo.md#alice-receives-credential
+But with all the auto-flags you can set on the agents, you dont have to do these steps yourself. But with the current configuration, no auto-flags, we will do those going forward
+
+## Defining the credential
+### Using v1.0
+Creating via the API, it gives you an example of attributes, they are explaned below.
+It is not possible to send in mismatching attributes, the agent will complain and let you know. Just test it out.
+There is 4 ways of doing this: **/create, /send, /send-proposal, /send-offer.** A questions is raised to learn what is the difference.
+For now we will move onwards with send
+
+#### Questions
+* What is the difference between send and create? Also, send-offer and send-proposal? Is it related to the auto flags or? Since doing send, it was not automated.
```

Comment:
`/send` (pretty sure...) does the entire flow without involving the controller until the issuance is completed. The others allow fine-grained control by the controller.

Reply:
@swcurran I was not able to experience this, after send was issued, I had to respond to the offer as well. So the questions might be, does send work, without the auto flags turned on? Or what is needed for this to work?

Or is it a multitenancy thing that it doesnt complete the whole process on /send?